assets/queries/openAPI/schema_default_invalid/query.rego

```diff
@@ -0,0 +1,20 @@
+package Cx
+
+import data.generic.openapi as openapi_lib
+
+CxPolicy[result] {
+	doc := input.document[i]
+	openapi_lib.check_openapi(doc) != "undefined"
+
+	[path, value] := walk(doc)
+
+	openapi_lib.invalid_field(value["default"], value.type)
```

Comment:
What happens if `default` is not defined?

Reply:
The query does not return any result